library/os_quota.py

```diff
@@ -0,0 +1,506 @@
+#!/usr/bin/python
+# Copyright (c) 2016 Pason System Corporation
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
So we don't account for files we've vendored in that are in a license other than Apache. Since we have a few files like this, we need to change the LICENSE or README.rst files to point out the files that are not our normal APACHE license. ( should not block this PR )